ft/Cargo.toml

```diff
@@ -8,5 +8,6 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-near-sdk = "3.1.0"
-near-contract-standards = "3.1.1"
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "1c5106bf0b3380d66d2689174f5e50b612101331" }
+near-contract-standards = { git = "https://github.com/near/near-sdk-rs", rev = "1c5106bf0b3380d66d2689174f5e50b612101331" }
```

Comment:
you can use `4.0.0-pre.7`. Events have been released with this version

Reply:
Done